src/login/popUp/PopUpLoginHandler.ts

```diff
@@ -48,32 +47,33 @@ export default class PopUpLoginHandler implements ILoginHandler {
     );
   }
 
-  async handle(loginOptions: ILoginOptions): Promise<ISolidSession> {
+  async handle(loginOptions: ILoginOptions): Promise<ISessionInfo> {
     const currentUrl = new URL(window.location.href);
     currentUrl.set("pathname", loginOptions.popUpRedirectPath);
     const session = await this.loginHandler.handle({
       ...loginOptions,
       redirect: currentUrl,
       doNotAutoRedirect: true
     });
+    return session;
     // TODO: handle if session doesn't have redirect
-    const popupWindow = window.open(
-      (session.neededAction as INeededRedirectAction).redirectUrl,
-      "Log In",
-      "resizable,scrollbars,width=500,height=500,"
-    );
-    return new Promise((resolve, reject) => {
-      const interval = setInterval(async () => {
-        if (!popupWindow || popupWindow.closed) {
-          clearInterval(interval);
-          resolve(
-            // TODO: handle if this fails
-            (await this.sessionCreator.getSession(
-              loginOptions.localUserId || "global"
-            )) as ISolidSession
-          );
-        }
-      }, 500);
-    });
+    // const popupWindow = window.open(
```

Comment:
Ditto

Reply:
Removed (and not implemented yet) in e59c56b09389ad1b96b748f905996bca79d78aeb